draft-ietf-taps-transport-security.md

```diff
@@ -159,14 +176,22 @@ efforts to define and catalog transport services {{RFC8095}} by describing the i
 add security protocols. It examines Transport Layer Security (TLS), Datagram Transport Layer 
 Security (DTLS), Quick UDP Internet Connections with TLS (QUIC + TLS), MinimalT, CurveCP, tcpcrypt, 
 Internet Key Exchange with Encapsulating Security Protocol (IKEv2 + ESP), SRTP (with DTLS), and 
-WireGuard. This survey is not limited to protocols developed within the scope or context of the IETF.
-
-For each protocol, this document provides a brief description, the security features it provides, 
-and the dependencies it has on the underlying transport. This is followed by defining the set of 
-transport security features shared by these protocols. Finally, we distill the application and 
+WireGuard. For each protocol, this document provides a brief description, the security features it 
+provides, and the dependencies it has on the underlying transport. This is followed by defining the 
+set of transport security features shared by these protocols. Finally, we distill the application and 
 transport interfaces provided by the transport security protocols. 
 
-Authentication-only protocols such as TCP-AO {{RFC5925}} and IPsec AH {{RFC4302}} are excluded
+Selected protocols represent a superset of functionality and features a TAPS system may 
+need to support, both internally and externally -- via an API -- for applications. Ubiquitous
```

Comment:
I may use parentheses here instead of dashes?

Reply:
Eh, parentheticals are often ignored.